### ISSUE TITLE
Remove treat_as_inputfiles for inputdata

### ISSUE THREAD

**Issue by egede** [ganga-devs/ganga#1600]
In the job class, there is in the `inputdata` field an option to give it an option of `treat_as_inputfiles`. This is a legacy option that gives rise to several other bugs. As the `treat_as_inputfiles` behaviour can be achieved with just using `inputfiles`, there is no reason to keep this legacy.

**Comment by varunbankar** [comment]
Should `inputdata.treat_as_inputfiles` be refactored to `inputfiles`?

**Comment by egede** [comment]
> Should `inputdata.treat_as_inputfiles` be refactored to `inputfiles`?

No, simpler than that. I think the option should be removed together with any code that deals with it. A starting point is given below. It seems like the use of the feature is in some tutorial, not suer which one though ...

```
(ganga) [egedelt] ~/programming/ganga/src/ganga % grep --include '*.py' -r treat_as_inputfiles
ganga/GangaCore/GPIDev/Lib/File/OutputFileManager.py:    if not job.subjobs and job.inputdata and isType(job.inputdata, GangaDataset) and job.inputdata.treat_as_inputfiles:
ganga/GangaCore/GPIDev/Lib/File/OutputFileManager.py:        if job.inputdata is not None and isinstance(job.inputdata, GangaDataset) and job.inputdata.treat_as_inputfiles:
ganga/GangaCore/GPIDev/Lib/File/OutputFileManager.py:        if job.inputdata and isType(job.inputdata, GangaDataset) and job.inputdata.treat_as_inputfiles:
ganga/GangaCore/GPIDev/Lib/File/OutputFileManager.py:        if job.master.inputdata and isType(job.master.inputdata, GangaDataset) and job.master.inputdata.treat_as_inputfiles:
ganga/GangaCore/GPIDev/Lib/Tasks/CoreTransform.py:                    unit.inputdata.treat_as_inputfiles = self.inputdata[
ganga/GangaCore/GPIDev/Lib/Tasks/CoreTransform.py:                        0].treat_as_inputfiles
ganga/GangaCore/GPIDev/Lib/Tasks/CoreTransform.py:        unit.inputdata.treat_as_inputfiles = self.chaindata_as_inputfiles
ganga/GangaCore/GPIDev/Lib/Dataset/GangaDataset.py:    schema['treat_as_inputfiles'] = SimpleItem(defvalue=False, doc="Treat the inputdata as inputfiles, i.e. copy the inputdata to the WN")
ganga/GangaCore/test/GPI/TutorialTests.py:        d.treat_as_inputfiles = True
ganga/GangaCore/Lib/Splitters/GangaDatasetSplitter.py:            j.inputdata.treat_as_inputfiles = job.inputdata.treat_as_inputfiles
ganga/GangaLHCb/Lib/LHCbDataset/LHCbDataset.py:    schema['treat_as_inputfiles'] = SimpleItem(defvalue=False, doc="Treat the inputdata as inputfiles, i.e. copy the inputdata to the WN")
ganga/GangaLHCb/Lib/LHCbDataset/LHCbCompressedDataset.py:    schema['treat_as_inputfiles'] = SimpleItem(defvalue=False, doc="Treat the inputdata as inputfiles, i.e. copy the inputdata to the WN")

```

**Comment by egede** [comment]
Fixed by #1603
